text/0000-add-status-field.md

```diff
@@ -0,0 +1,86 @@
+# Meta
+[meta]: #meta
+- Name: Add Status to RFC Metadata
+- Start Date: 2021-07-14
+- Author(s): @aemengo, @ekcasey
+- Status: Draft <!-- Acceptable values: Approved, On Hold, Draft -->
```

Comment:
Should we add  "Superseded" to the list of statuses? An RFC PR that includes RFCs in the "Supersedes" section should also changed the status of all referenced RFCs to "Superseded".
```suggestion
- Status: Draft <!-- Acceptable values: Approved, Draft, On Hold, Superseded -->
```

(I also alphabetized the statuses in this suggestion b/c I couldn't see the logic in the original ordering)

Reply:
Is there value of having a "Draft" status? We don't have reject here.

Reply:
> Is there value of having a "Draft" status? 

If someone links you to a readable version of a not-yet-merged RFC, you can see that it is still a draft.

Reply:
Should we include closed/postponed as well then? Is there a distinction between "status" and "disposition"?

Reply:
I originally called this out as an anti-goal; trying to show all the status that could be inferred by the PR process. Don't know how you still feel about closed/postponed